cibyl/models/ci/zuul/job.py

```diff
@@ -123,6 +123,15 @@ def __eq__(self, other):
 
     API = {
         **subset(BaseJob.API, ['name', 'url', 'builds']),
+        'output_mode': {
+            'arguments': [
+                Argument(
+                    name='--complete', arg_type=bool, nargs=0,
```

Comment:
As mentioned in a previous comment, I think that this functionality should come via a combination of the --tenants, --pipelines and --verbose arguments, rather than introducing a new one